fast_api_als/routers/three_pl.py

```diff
@@ -17,9 +17,11 @@ async def reset_authkey(request: Request, token: str = Depends(get_token)):
     body = json.loads(body)
     provider, role = get_user_role(token)
     if role != "ADMIN" and (role != "3PL"):
+        logging.error("Invalid user")
```

Comment:
raise http exception with proper status code

Reply:
Resolved

Reply:
Improved the http exceptions with proper status codes in all files, please check now